apiary-ranger-metastore-plugin/src/main/java/com/expediagroup/apiary/extensions/rangerauth/listener/ApiaryRangerAuthPreEventListener.java

```diff
@@ -83,13 +83,13 @@ public ApiaryRangerAuthPreEventListener(Configuration config) throws HiveExcepti
     plugin = new RangerBasePlugin("hive", "metastore");
     plugin.init();
     plugin.setResultProcessor(new RangerDefaultAuditHandler());
-    log.debug("ApiaryRangerAuthPreEventListener created");
+    log.info("ApiaryRangerAuthPreEventListener created");
```

Comment:
Do these really need to be on info? I'd prefer to keep this kind of normal expected behaviour on `debug` so the logs don't get too noisy.

Reply:
see my other comment on why I think we are far too opposed to INFO level logs.